.php-cs-fixer.dist.php

```diff
@@ -17,21 +17,64 @@
     ->setRules([
         '@PSR12' => true,
         'array_syntax' => ['syntax' => 'short'],
+        'backtick_to_shell_exec' => true,
         'braces' => [
             'allow_single_line_closure' => true,
             'allow_single_line_anonymous_class_with_empty_body' => true,
         ],
+        'class_definition' => ['single_line' => true],
         'concat_space' => ['spacing' => 'one'],
         'declare_strict_types' => true,
+        'elseif' => true,
+        'encoding' => true,
+        'ereg_to_preg' => true,
+        'explicit_string_variable' => true,
+        'fully_qualified_strict_types' => true,
         'function_typehint_space' => true,
+        'general_phpdoc_annotation_remove' => [
+            'annotations' => [
+                'author',
+                'package',
+                'subpackage',
+                'version',
+            ],
+        ],
+        'global_namespace_import' => [
+            'import_functions' => true,
+        ],
+        'include' => true,
+        'increment_style' => true,
```

Comment:
Shouldn't the value be "pre" or "post"? 
https://mlocati.github.io/php-cs-fixer-configurator/#version:3.8|fixer:increment_style

Reply:
By default is 'pre', but I can change it for clarity to
```php
'increment_style' => ['style' => 'pre'],
```